prime-router/settings/organizations.yml

```diff
@@ -2513,6 +2519,26 @@
         port: 22
         filePath: ./upload
         credentialName: DEFAULT-SFTP
+    - name: HL7_MONKEYPOX
+      externalName: Ignore Monkeypox
+      organizationName: ignore
+      topic: monkeypox
+      customerStatus: inactive
+      jurisdictionalFilter: [ "matches(ordering_facility_county, HL7_MONKEYPOX)" ]
+      timing:
+        operation: MERGE
+        numberPerDay: 1440 # Every minute
+        initialTime: 00:00
+      translation:
+        type: CUSTOM
+        schemaName: monkeypox
+        format: HL7
+      transport:
+        type: SFTP
+        host: sftp
+        port: 22
+        filePath: ./upload
+        credentialName: DEFAULT-SFTP
```

Comment:
See above. Setting up a hMPXv sender and receiver in IGNORE for future testing work.